.github/workflows/maven.yml

```diff
@@ -0,0 +1,62 @@
+name: Maven CI Build
+
+on: [push]
+
+jobs:
+  JDK6Toolchain:
+    name: Toolchain 1.6, JDK 11, OS ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, ubuntu-20.04, windows-2019, windows-2022]
+      fail-fast: true
+      max-parallel: 2
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install Toolchain JDK
+        uses: battila7/jdk-via-jabba@v1
+        with:
+          jdk: zulu@1.6.119
```

Comment:
consider zulu@1.7.292 instead?

Reply:
Yup...and Log4j 1 supports JDK 1.4 and up! Details in PR discussion.